packages/analytics-core/src/event-bridge/event-receiver.ts

```diff
@@ -0,0 +1,24 @@
+import { Config, CoreClient, Event, EventBridgeReceiver } from '@amplitude/analytics-types';
+import { EventChannel } from './event-channel';
+
+export class AnalyticsEventReceiver implements EventBridgeReceiver {
```

Comment:
Is this now used anywhere? Or this is just about the module?

Reply:
Not used since yet. Since it's not being used yet, I'd prefer not to include it in the bundle yet. Maybe we can include in the lazy loading strategy we previously talked about.